### PR TITLE
CLOUD-42 Add proper Payara branding to the UI, as according to the design doc

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Not yet complete

File: deployer/deployment-controller/src/main/webapp/main.css [hunk 1]
```diff
@@ -93,6 +105,10 @@ h1 a {
     color: #0096d6;
 }
 
+img {
```
Comment (pdudits) [inline deployer/deployment-controller/src/main/webapp/main.css]: This looks bit general. Did you mean `img.logo`?

File: deployer/deployment-controller/src/main/webapp/index.html [hunk 2]
```diff
@@ -46,21 +46,24 @@
...
+                <p><label for="project">Project <sup>(required)</sup></label><a class="hint">?</a></p>
                 <input id="project" name="project" required type="text" pattern="[a-zA-Z0-9-]{4,30}" placeholder="DNS-like name, at least 4 characters (required)">
-                <label for="stage">Stage</label>
+                <p><label for="stage">Stage <sup>(required)</sup></label><a class="hint">?</a></p>
                 <input id="stage" name="stage" required type="text" pattern="[a-zA-Z0-9]{1,30}" placeholder="DNS-like name, at least one character">
-                <label for="artifact">Artifact</label>
-                <input type="file" accept=".war" id="artifact" name="artifact">
-                <input type="submit" value="Deploy">
+                <p><label for="artifact">Artifact <sup>(required)</sup></label><a class="hint">?</a></p>
+                <input type="file" accept=".war" id="artifact" name="artifact" class="button" required multiple/>
```
Comment (pdudits) [inline deployer/deployment-controller/src/main/webapp/index.html]: We don't support multiple uploads

File: deployer/deployment-controller/src/main/webapp/main.css [hunk 3]
```diff
@@ -117,9 +168,82 @@ progress {
...
+
+h3 {
+    font-size: 1.25rem;
+}
+
+h4, h5, h6 {
+    font-size: 1rem;
+}
+
+checkbox, radio {
```
Comment (pdudits) [inline deployer/deployment-controller/src/main/webapp/main.css]: Do you mean `input[type=checkbox], input[type=radio]`?

**Pull request merged**